pkgs/development/compilers/dotnet/combinePackages.nix

```diff
@@ -0,0 +1,20 @@
+{ packages }: 
+{ buildEnv, lib }:
+let cli = builtins.head packages;
+in
+assert lib.assertMsg ((builtins.length packages) != 0) 
+    ''You must include at least one package, e.g 
+      `with dotnetCorePackages; combinePackages {
+          packages = [ sdk_3_0 aspnetcore_2_1 ];
+       };`'' ;
```

Comment:
```

    ''You must include at least one package, e.g 
      `with dotnetCorePackages; combinePackages [ sdk_3_0 aspnetcore_2_1 ];`
    '' ;
```

Reply:
Thank you for the suggestions. I have updated the commits